.github/workflows/ci.yml

```diff
@@ -95,7 +95,7 @@ jobs:
           - ubuntu18
           - ubuntu20
           - redhat8
-        suite: [ 'base' ]
+        suite: [ 'sudo', 'users' ]
```

Comment:
What happens if you remove this line? Does it execute all the suites or nothing?

Reply:
removed, all of them are executed.